docs/schemaActions/README.md

```diff
@@ -1,27 +1,101 @@
+be submitted to the developers.
 
 ### Outcome
-The majority of schema action structures include an `<Outcome>` element. This is used to described whether the action was successful/permitted or not, e.g. a failed logon or a user being denied access to a file.
+The majority of schema action structures include an `<Outcome>` element. This is used to described whether the action
+was successful/permitted or not, e.g. a failed logon or a user being denied access to a file.
 
-`<Success>` is used to describe whether something failed or went wrong in some way, e.g. an export to removable media that failed due to lack of space. This element is used in cases of unexpected failure rather than where something has been prevented due to some rule or process, where `<Permitted>` would be more appropriate.
+`<Success>` is used to describe whether something failed or went wrong in some way, e.g. an export to removable media
+ that failed due to lack of space. This element is used in cases of unexpected failure rather than where something 
+ has been prevented due to some rule or process, where `<Permitted>` would be more appropriate.
 
-`<Permitted>` is used to describe whether an action was prevented due to some form of authorisation or rule, e.g. a user not being permitted access to a document due to having insufficient permissions.
+`<Permitted>` is used to describe whether an action was prevented due to some form of authorisation or rule,
+ e.g. a user not being permitted access to a document due to having insufficient permissions.
 
 In both cases a value of true is assumed so the element only needs to be included if the action was unsuccessful.
 
-The `<Description>` element should be used to described the detail around the failure of the action as this will be useful to a human looking at the event.
+The `<Description>` element should be used to described the detail around the failure of the action as this
+ will be useful to a human looking at the event.
 
 ### Data
-This provides a flexible structure for describing aspects of the schema action that cannot be described using the structure in the schema. This structure is described in more detail in [Unstructured Data](../unstructuredData.md)
+This provides a flexible structure for describing aspects of the schema action that cannot be described using
+ the structure in the schema. This structure is described in more detail in [Unstructured Data](../unstructuredData.md)
+
+The majority of the 'schema action' elements include the MultiObjectComplexType structure (or some child of it)
+ to allow for recording various forms of objects that were involved in the event.
+
+##Defined Schema Actions
+A number of schema actions are defined that aim to be broadly representative of the kinds of activity that are 
+most likely to be of interest from an audit point of view.
+
+Schema Action|Rough Description |
+|---|---|
```

Comment:
pad out the table cells to make it more readable in raw form

something like this is your firend - https://robots.thoughtbot.com/align-github-flavored-markdown-tables-in-vim, other plugins are available ;-)

Reply:
fixed in #26